pkg/storaged/client.js

```diff
@@ -17,19 +17,28 @@
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
 
-define([
-    "jquery",
-    "base1/cockpit",
-    "storage/utils",
-    "manifests"
-], function($, cockpit, utils, manifests) {
+(function() {
+    "use strict";
+
+    var $ = require('jquery');
+    var cockpit = require('cockpit');
+
+    var utils = require('./utils');
 
     /* STORAGED CLIENT
      */
 
+    /* HACK: https://github.com/storaged-project/storaged/pull/68 */
```

Comment:
It's related. The whole way the manifests are used here is a HACK around storaged. I just added the comment to clarify what's going on. Should I put the comment in a separate pull request. In any case I'll rebase this onto #4915 and that should help this work better.